.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+indent_size = 4
+end_of_line = lf
+
+[*.{c,h,cxx,am,m4}]
+indent_style = tab
```

Comment:
it should be gobal.  When we edit Geany in Geany all filetypes should get indent 4 and the idea is to make github look the same.